.github/ISSUE_TEMPLATE/documentation.yml

```
@@ -0,0 +1,30 @@
+name: Documentation
+description: Report problems and suggest ideas for the documentation
+labels: [docs]
+
+body:
+  - type: markdown
+    attributes:
+      value: Before opening a new issue, please search for duplicate or closed issues.
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: Skip to "other" if you do not see your issue type here.
+      options: [improvement, mistake]
+
+  - type: input
+    attributes:
+      label: Kind of issue (other)
+      description: Only fill out if your issue type is not in the dropdown above.
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Anything relevant, for example:
+        - A (perma)link to the documentation
+        - For improvements: Suggested for how to improve
+        - For mistakes: Suggested fixes
```

Not really your "fault", but I think the description can use some improvements.

```suggestion
        Include as much detail as possible, for example:
        - A (perma)link to the documentation in question
        - Suggestions for how to change the docs
```

Other suggestions are welcome

---

There really isn't much to "documentation" issues, so I think this should be fine.